org.eclipse.jdt.ui.tests.refactoring/.classpath

```diff
@@ -6,10 +6,10 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="con" nonaccessible="org/eclipse/core/internal/**|org/eclipse/ui/internal/**" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="test cases">
+	<classpathentry kind="src" path="test cases/">
```

Comment:
Please explain this change in your commit comment or remove it if it isn't really part of enabling jdt.ls.